blog/2015/11/python-lesson-rewrite.html

```diff
@@ -0,0 +1,104 @@
+---
+layout: blog
+root: ../../..
+author: Matt Davis
+title: "Python Lesson Rewrite"
+date: 2015-11-14
+time: "11:30:00"
+category: ["Lessons", "Python"]
+---
+<!-- start excerpt -->
+<section>
+  <p>
+    Recent
+    <a href="http://software-carpentry.org/blog/2015/09/thinking-about-teaching.html">
+      analysis
+    </a>
+    and
+    <a href="http://software-carpentry.org/blog/2015/10/python-debriefing-summary.html">
+      introspection
+    </a>
+    regarding Software Carpentry's Python lesson has led us to conclude that
+    the lesson would benefit from a complete rewrite.
+    The subcommittee on lesson development
+    <a href="http://pad.software-carpentry.org/python-lesson-planning-2015-10-14">met twice</a>
+    to plan much about the new lessons (yes, that's plural).
+    This post will describe our plans for the new lessons.
+  </p>
+  <p>
+    We're going to write two new Python lessons, one intended for students who
+    are completely new to programming (the "novice" lesson),
+    and another for students with some programming experience (the "intermediate" lesson).
+    <a href="http://www.datacarpentry.org/">Data Carpentry</a> also targets
+    new programmers so we'll be developing the new novice lesson in collaboration
+    with DC, and both DC and SWC will use the lesson.
+    We decided to use <a href="http://www.gapminder.org/">Gapminder</a> data
+    for both the novice and intermediate lessons in order to have a
+    consistent experience and approachable, meaningful data
+    (and because some lessons are already using it).
+  </p>
+</section>
+<!-- end excerpt -->
+<section>
+  <h2>Novice Lesson</h2>
+  <p>
+    The overarching goal of the novice lesson is to convince students to adopt
+    programming in their everyday work flow.
+    To do that we want to both teach practical data tools
+    and demonstrate the power of writing code (instead of using Excel).
+    Our lesson will include using <a href="http://pandas.pydata.org/">Pandas</a>
+    to work with data and using Python's
+    <a href="https://docs.python.org/3/tutorial/controlflow.html">control flow features</a>
+    to really make the computer work.
+  </p>
+  <p>
+    Greg Wilson will be leading the development of the new novice lesson.
```

Comment:
Do we want to have a DC contact here too since we're saying we're developing in conjunction with DC?